lib/zendesk_apps_support/package.rb

```diff
@@ -289,34 +299,35 @@ def location_icons
 
           host = location_options.location.product.name
           location = location_options.location.name
-          location_icons[host][location] = build_location_icons_hash(location)
+          product_prefix = manifest.products.count > 1 ? "#{host}/" : ''
+          location_icons[host][location] = build_location_icons_hash(location, product_prefix)
         end
       end
     end
 
-    def build_location_icons_hash(location)
+    def build_location_icons_hash(location, product_prefix)
```

Comment:
nit: I'm not sure `product_prefix` makes sense as an argument name here - it's in the middle of the icon url, after all, not at the start. Maybe `product_dir`?

Reply:
Yeah good call 👍 I'll go with `product_directory`